arch/arm/src/gd32f4/gd32f4xx_syscfg.c

```diff
@@ -107,7 +107,7 @@ void gd32_syscfg_exmc_swap_config(uint32_t syscfg_exmc_swap)
 void gd32_syscfg_exti_line_config(uint8_t exti_port, uint8_t exti_pin)
 {
   uint32_t regval;
-  uint32_t regaddr;
+  uint32_t regaddr = 0;
```

Comment:
Let's also add `DEBUGASSERT(false);` in the `default` case?

Reply:
Done.